web-ui/src/common/header/header.js

```diff
@@ -26,14 +27,11 @@ export const Header = ({ t }) => (
         <img
           className='header-logo'
           src='/public/images/logo-orange.svg'
-          alt='Pixelated'
+          alt={t('Pixelated')}
```

Comment:
Is there a reason we decided to translate "Pixelated"? 😆 I guess the questions is actually: we are saying we're going to translate it here, but there's actually no key for "Pixelated" in the translations json. This will most likely fail the integrations tests.